source/Hyperspace-Model/MultiValueOrderedDictionary.class.st

```diff
@@ -0,0 +1,40 @@
+"
+I'm like ZnMultiValueDictionary but preserving the insertion order of my keys.
```

Comment:
Is the reference to ZnMultiValueDictionary outdated?

Reply:
No. This class is akin ZnMultiValueDictionary but preserving the insertion order. That way, the tests are deterministic.